### PR TITLE
Remove compare

### PR DESCRIPTION
**Pull request by georgiana-b**
This PR refactors the code to remove the compare module, air data and anything related to that.

The relevant logic has been incorporated in the stats module.

Everything else has been removed.

This shouldn't have any functional effect on the dashboard page.

### CODE REVIEW

File: edscrapers/tools/dashboard/pages/insights.py [hunk 1]
```diff
@@ -171,27 +173,30 @@ def resources_by_domain_table(self):
...
-        #df = self._get_df_from_excel_sheet('RESOURCE COUNT PER DOMAIN (DATOPIAN ONLY)')
-
-        #working_df2 = pd.DataFrame(columns=['domain'])
-        #working_df2['domain'] = df['domain']
-        #working_df2['resource count'] = df['resource count']
-
-        # concatenate the 2 dataframes
-        #working_df1 = pd.concat([working_df1,
-        #                        working_df2], axis='index', ignore_index=True)
+        # df = self.resources_by_domain_df()
```
Comment (georgiana-b) [inline edscrapers/tools/dashboard/pages/insights.py:176]: This is the refactored code but I had to comment it to comply with the changes Higor made.

File: edscrapers/tools/dashboard/pages/insights.py [hunk 2]
```diff
@@ -148,8 +149,9 @@ def dataset_by_domain_bar(self):
         """ function creates a bar chart which displays the
         number of pages/datasets per domain """
         # the the dataframe from the Excel sheet
-        #df = self._get_df_from_excel_sheet('PAGE COUNT (DATOPIAN)')
+        #df = self._get_df_from_excel_sheet('PAGE COUNT')
```
Comment (georgiana-b) [inline edscrapers/tools/dashboard/pages/insights.py:152]: This is the refactored code but I had to comment it to comply with the changes Higor made

File: edscrapers/tools/dashboard/json_parser.py [hunk 3]
```diff
@@ -62,7 +60,7 @@ def get_total_resources_by_office(source, is_sorted=True):
         data = data['total'][source]['resources_by_office'].items()
         return dict(sorted(data, key=lambda item: item[1], reverse=True))
 
-def get_total_pages_by_office(source, is_sorted=True):
+def get_total_datasets_by_office(source, is_sorted=True):
```
Comment (nightsh) [inline edscrapers/tools/dashboard/json_parser.py:63]: If the return value is pages, why the function name change?

File: edscrapers/tools/stats/stats.py [hunk 4]
```diff
@@ -11,10 +12,20 @@
...
+    'edgov': r'www2.ed.gov',
+    'octae': r'ovae|OVAE|octae|OCTAE',
+    'oela': r'oela|ncela',
+    'ope': r'/ope/',
+    'opepd': r'\bopepd\b',
+    'osers': r'osers|osep|idea',
+    'ocr': r'ocrdata.ed.gov',
+    'oese': r'oese.ed.gov',
+    'nces': r'\bnces\b',
+}
```
Comment (nightsh) [inline edscrapers/tools/stats/stats.py:27]: Add `fsa`. We need to think of a better way of doing this.

File: edscrapers/tools/dashboard/json_parser.py (hunk 3 above)
Comment (nightsh) [reply edscrapers/tools/dashboard/json_parser.py:63]: Also, this doesn't seem to be used, will flag in the code (mark for review later on).

**Pull request merged**